content/docs/tasks/traffic-management/egress/egress-gateway/index.md

```diff
@@ -776,6 +776,24 @@ external service.
     cluster.outbound|443||edition.cnn.com.upstream_cx_total: 2
     {{< /text >}}
 
+1.  If you're using a `SOURCE_POD` that is running in a different namespace than the one where you applied the
```

Comment:
I am afraid this part will over-complicate the task. How about a warning in the beginning of the task: _create the configuration items below in the same namespace you run SOURCE_POD_. Or _the instructions below assume that you run SOURCE_POD in the default namespace_.

Reply:
Yeah, I wasn't sure about this either. I want to mention something, because this is the problem that the user ran into in https://github.com/istio/istio/issues/15556.